src/clr-angular/layout/grid-deprecated.clarity.scss

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+// @deprecated Bootstrap dependencies are deprecated since 1.0. Remove before 2.0.
+
+$enable-grid-classes: true;
+$imported-modules: () !default;
+
+@mixin exports($name) {
```

Comment:
i thought `exports` was something we used everywhere. are we re-defining it here? or did we pull it out by accident?

Reply:
Its defined in the `_mixins.clarity.scss`. I was trying to keep this deprecated sctylesheet as light as possible. You are right though and I'll just include the global mixins.